coast/DOMAIN.py

```diff
@@ -110,3 +112,33 @@ def subset_indices_by_distance(self, centre_lon, centre_lat, radius):
         # Then these output tuples can be separated into x and y arrays if necessary.
 
         return nemo_indices
+
+    def find_J_I(self, lat, lon, grid_ref: str):
+        """
+            Simple routine to find the nearest J,I coordinates for given lat lon
+            Usage: [J,I] = findJI(49, -12, nav_lat_grid_T, nav_lon_grid_T)
+        """
+
+        interal_lat = f"gphi{grid_ref}"
+        interal_lon = f"glam{grid_ref}"
+        dist2 = xa.ufuncs.square(self.dataset[interal_lat] - lat) + xa.ufuncs.square(self.dataset[interal_lon] - lon)
+        [t, y, x] = np.unravel_index(dist2.argmin(), dist2.shape)
```

Comment:
If t isn't going to be returned, it should be replaced with _ to denote an unused variable.